README.md

```diff
@@ -1,697 +1 @@
-  1. [Functions](#functions)
-  1. [ReactJs](#reactjs)
-
-## General
-
-  - Console*[no-console]: console.log should be removed whenever the code will be pushed
-  - Always use `isNaN()` function when checking if variable is NaN[use-isnan]
-
-  ```js
-  // BAD
-  if (x === NaN) { ... }
-
-  // GOOD
-  if (isNaN(x)) { ... }
-  ```
-
-  - No unused variable[no-unused-var]
-  - Always use **2 spaces** for idention and not tabs[no-mixed-spaces-and-tabs] [no-tabs]
-  - No variable assignment on return statemetns[no-return-assign]
-
-  ```js
-  // BAD
-  function fn () {
-    return a = 2 + 2
-  }
-  ```
-
-  - No script for URLS[no-script-url]
-
-  ```js
-  // BAD
-  location.href = 'javascript:void(0)'
-  ```
-
-  - I'm sorry yoda but you are not allowed[yoda]
-
-  ```js
-  // BAD
-  if (true === foo) { ... }
-  if ('test' === bar) { ... }
-
-  // GOOD
-  if (foo === true) { ... }
-  if (bar === 'test') { ... }
-  ```
-
-  - Variable names should be camelCased except properties*[camelcase]
-
-  ```js
-  // BAD
-  const not_camelcased = 'NOOOO!'
-  import { no_t_camel } from 'bar'
-
-  // GOOD
-  const camelCase = 'YEY'
-  import { camelCased } from 'foo'
-  obj = {
-    test_me: 5
-  }
-  ```
-
-  - There should be no commas on the last element of an array or object[comma-dangle]
-
-  ```js
-  // BAD
-  const badArray = [
-    foo,
-    bar,
-  ]
-  const badObj = {
-    foo: 'test',
-    bar: 'me',
-  }
-
-  // GOOD
-  const goodArr = [
-    foo,
-    bar
-  ]
-  const goodObj = {
-    foo: 'test',
-    me: 'bar'
-  }
-  ```
-
-## Conditional Statement
-
-  - No constant expressions in conditions[no-constant-condition]
-
-  ```js
-  // BAD
-  if (false) {
-    doSomething()
-  }
-
-  while (true) {
-    doSomething()
-  }
-  ```
-
-  - No empty block statements[no-empty]
-
-  ```js
-  // BAD
-  if (foo) {
-  }
-
-  While () {
-  }
-
-  // GOOD
-  if (wat) {
-    // Comment here
-  }
-  ```
-
-  - No case statement fallthrough[no-fallthrough]
-
-  ```js
-  // BAD
-  switch (f) {
-    case 1:
-      doSomething()
-    case 2:
-      doAnything()
-  }
-
-  // GOOD
-  switch (f) {
-    case 1:
-      doSomething()
-      break;
-    case 2:
-      return doAnything()
-    case 3:
-      ...
-  }
-
-  switch (f) {
-    case 1:
-    case 2:
-      return doForBoth()
-  }
-  ```
-
-## Whitespace
-
-  - No irregular whitespace[no-irregular-whitespace]
-
-  ```js
-  // BAD
-  function test (  test) {
-    return 'test'
-  }
-  ```
-
-  - There should be no spacing on array brackets except for deconstructing array[array-bracket-spacing]
-
-  ```js
-  // BAD
-  const a = [ 'test' ]
-  const b = [ 'foo', 'arf']
-  var [x,y,x] = a
-
-  // GOOD
-  const a = ['boo']
-  const c = ['foo', 'bar']
-  var [ x, y, z ] = a
-  ```
-
-  - Spacing on commas should be after the commas[comma-spacing]
-
-  ```js
-  // BAD
-  var arr = [ 1 ,2 ]
-  var c = [ 1 , 2 ]
-
-  // GOOD
-  var b = [ 2, 3 ]
-  ```
-
-  - Indention on blocks should be 2 spaces[indent]
```

Comment:
Then I don't see your problem. That's like the most normal thing 
:baby: :boy: :man:
:eye: :zap: :woman:
:couple: :cupid: :timer_clock: :baby: :family:
:timer_clock: :older_man: :older_woman: :skull: :ghost: